great_expectations/jupyter_ux/expectation_explorer.py

```diff
@@ -1003,7 +1004,7 @@ def generate_parse_strings_as_datetimes_widget_dict(
         **expectation_kwargs,
     ):
         data_asset_name = expectation_state["data_asset_name"]
-        data_asset = self.state["data_assets"].get(data_asset_name)["data_asset"]
+        data_asset = self.state["data_assets"][data_asset_name]["data_asset"]
```

Comment:
run-time changes above, shouldn't change anything assuming we aren't counting on `AttributeError: 'NoneType' object has no attribute 'something'` exceptions to drive control flow 😄 .